### PR TITLE
Change from settlementAccountNumnber to bankAccount

### PR DESCRIPTION
**Pull request by niklud**

### CODE REVIEW

**Review by cloveras — APPROVED** [review]

File: management-api-guide.md [hunk 1]
```diff
@@ -289,9 +291,9 @@ Here is a sample request to
   },
   "salesUnitName": "ACME Fantastic Fitness",
   "salesUnitLogo": "VGhlIGltYWdlIGdvZXMgaGVyZQ==",
-    "settlementAccountNumber": {
-    "scheme": "BBAN:NO",
-    "id": "86011117947"
+    "bankAccount": {
```
Comment (cloveras) [inline management-api-guide.md:294]: The Report API uses `payoutBankAccount`, so I think we should prefix the account here too, and keep `settlementAccount`.

Report API: https://developer.vippsmobilepay.com/api/report/#/paths/~1settlement~1v1~1ledgers/get

Comment (niklud) [reply management-api-guide.md]: Like this? [91a1481](https://github.com/vippsas/management-api/pull/45/commits/91a1481cd8fd45a463d92d974450d4c1a15150a4)

File: management-api-guide.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ Here is a sample request to
   },
   "salesUnitName": "ACME Fantastic Fitness",
   "salesUnitLogo": "VGhlIGltYWdlIGdvZXMgaGVyZQ==",
-    "bankAccount": {
+    "settlementAccount": {
```
Comment (cloveras) [inline management-api-guide.md:294]: settlementBankAccount, please (I can't suggest code here)

File: management-api-guide.md (hunk 1 above)
Comment (cloveras) [reply management-api-guide.md]: So close!

File: docs/swagger.yaml [hunk 3]
```diff
@@ -588,8 +588,8 @@ components:
             The minimum size is 100 x 100 pixels.
             Images not matching the above criteria will not be shown to the merchant in the merchant portal.
             The logo can be edited later on portal.vipps.no.
-        settlementAccountNumber:
-          $ref: '#/components/schemas/bankAccountBBAN'
+        settlementAccount:
```
Comment (cloveras) [inline docs/swagger.yaml:591]:
```suggestion
        settlementBankAccount:
```

File: management-api-guide.md [hunk 4]
```diff
@@ -289,9 +291,9 @@ Here is a sample request to
   },
   "salesUnitName": "ACME Fantastic Fitness",
   "salesUnitLogo": "VGhlIGltYWdlIGdvZXMgaGVyZQ==",
-    "settlementAccountNumber": {
-    "scheme": "BBAN:NO",
-    "id": "86011117947"
+    "settlementAccount": {
```
Comment (cloveras) [inline management-api-guide.md:294]:
```suggestion
    "settlementBankAccount": {
```

File: management-api-guide.md (hunk 1 above)
Comment (niklud) [reply management-api-guide.md]: I see your edit in the first post there, no sneaking that past me, your fault! :P

**Pull request merged**